src/cryptography/hazmat/primitives/hashes.py

```diff
@@ -136,6 +136,34 @@ class SHA512(object):
     block_size = 128
 
 
+@utils.register_interface(HashAlgorithm)
+class SHA3_224(object):  # noqa: N801
+    name = "sha3-224"
+    digest_size = 28
+    block_size = None
```

Comment:
block_size isn't really a relevant value for a sponge function. It has a state size, but not a block.

Reply:
We removed block_size from the interface I thought? This shouldn't be a problem